File: src/_1ProgrammingBasicsWithJavaMay2023/_03ConditionalStatementsAdvanced/_1Lab/_03AnimalType.java
package _1ProgrammingBasicsWithJavaMay2023._03ConditionalStatementsAdvanced._1Lab;

import java.util.Scanner;

public class _03AnimalType {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String animal = scanner.nextLine();
        switch (animal) {
            case "dog":
                System.out.println("mammal");
                break;
            case "crocodile":
            case "tortoise":
            case "snake":
                System.out.println("reptile");
                break;
            default:
                System.out.println("unknown");
                break;
        }
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_3ConditionalStatementsAdvanced/_1Lab/_03AnimalType.js
function animalType(animalType) {
    let animalClass = "";

    if (animalType === "dog") {
        animalClass = "mammal";
    } else if (animalType === "crocodile") {
        animalClass = "reptile";
    } else if (animalType === "tortoise") {
        animalClass = "reptile";
    } else if (animalType === "snake") {
        animalClass = "reptile";
    } else {
        animalClass = "unknown";
    }

    console.log(`${animalClass}`)
}